src/transformers/models/t5/modeling_t5.py

```diff
@@ -1447,6 +1479,18 @@ def forward(
         use_cache = use_cache if use_cache is not None else self.config.use_cache
         return_dict = return_dict if return_dict is not None else self.config.use_return_dict
 
+        # FutureWarning: head_mask was separated into two input args - head_mask, decoder_head_mask
+        if head_mask is not None and decoder_head_mask is None:
+            if self.config.num_layers == self.config.num_decoder_layers:
+                warning_msg = """
```

Comment:
I see this message is used twice, maybe it could be refactored in a private constant?

Reply:
That's a good point, thanks!